.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: weekly
+    ignore:
+      - update-types: ["version-update:semver-patch", "version-update:semver-minor"]
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    ignore:
+      - update-types: ["version-update:semver-patch", "version-update:semver-minor"]
```

Comment:
Nit: fine to leave the minor patches in for GH actions I think, the config only ever contains `@vX` with just the major version.